create_book.rb

```diff
@@ -0,0 +1,15 @@
+class CreateBook
+  def initialize(book)
+    @books = book
+  end
+
+  def create_book
+    print 'Book Title:'
+    title = gets.chomp
+    print 'Book Author:'
+    author = gets.chomp
+    new_book = Book.new(title, author)
+    @books.push(new_book)
+    puts "#{new_book.title} created successfully"
+  end
```

Comment:
- [ ] Please it would be better if you separate all the requests for input and input validation into private methods this way you achieve single responsibility principle

Reply:
@youmari please can you explain what you mean.

Reply:
@Kamba56 what I meant is make sure any code that takes answers from the users or validates the answer should be separate from creating book and put them in a private methods